docs/admin/troubleshooting_network.rst

```diff
@@ -2,3 +2,218 @@ Troubleshooting network issues
 ==============================
 
 .. include:: ../includes/top-warning.rst
+
+Before troubleshooting network issues, we recommend reading about the
+:ref:`networking architecture <Networking Architecture>`
+of SecureDrop Workstation. If you are in a hurry, this guide offers quick
+diagnostic and remedial steps.
+
+Step 1: Verify you are connected to the Internet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use both wireless and wired networks in Qubes. You can manage network
+access through the network manager, which you can find in the area populated
+with icons in the top right corner of your Qubes desktop, known as the *system
+tray*.
+
+The network manager looks like this for a wired connection:
+
+**[SCREENSHOT: network manager wired icon]**
+
+It looks like this for a wireless connection:
+
+**[SCREENSHOT: network manager wired icon]**
+
+It looks like this when you are not connected to the Internet at all:
+
+**[SCREENSHOT: no connection icon]**
+
+When a network connection is lost, Qubes will display an alert like the
+following:
+
+**[SCREENSHOT: lost connection notification]**
+
+Common causes for lost connections include fully or partly unplugged network cables,
+lost power to networking equipment, and ISP service outages. When you see a lost
+connection notification, it is most likely due to one of these causes.
+
+If the network manager shows that you are connected to the Internet, you can
+verify whether your connection is working by opening a terminal in ``sys-net``:
+
+**[SCREENSHOT: Q widget with VM list and "Run terminal" expanded]**
+
+1. Click the "Q" icon in the in the system tray (top right area).
+2. A list of running VMs should appear. Select ``sys-net`` from the list, and
+   click **Run Terminal**.
+3. In the terminal window, type the command ``ping google.com``.
```

Comment:
Maybe `ping 8.8.8.8` to avoid DNS gotchas? :thinking:

Reply:
I think that's a great additional step we can suggest here, will add.